cmd/boulder-ca/main.go

```diff
@@ -56,10 +59,16 @@ type Config struct {
 		// The maximum number of subjectAltNames in a single certificate
 		MaxNames int
 
-		// LifespanOCSP is how long OCSP responses are valid for; It should be longer
-		// than the minTimeToExpiry field for the OCSP Updater.
+		// LifespanOCSP is how long OCSP responses are valid for. It should be
```

Comment:
Love the addition of this comment; I took a look at the logic in main where this is loaded from config and the constructor to which the parsed duration is passed I don't see anywhere where we validate this.